app/models/game.rb

```diff
@@ -14,6 +14,30 @@ def populate!
     populate_right_white_half!
   end
 
+  def white_check?
```

Comment:
look at the parts of your code that are repetitive and think of ways to refactor using helper methods to resolve the problem of method being too long for rubocop